regen-network/regen-web
# feat: APP-23 project data stream

## Description

https://regennetwork.atlassian.net/browse/APP-23?atlOrigin=eyJpIjoiMDQ2MDQ0OTFkNWU1NDI1MDg4ZTZkZDNlMzA3M2U3ZWMiLCJwIjoiaiJ9

I've added 2 new tasks for:
- post deletion: https://regennetwork.atlassian.net/browse/APP-188
- sharing private link: https://regennetwork.atlassian.net/browse/APP-189

which are not included in this work because it needs some larger server related work.



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test

1. Check out https://deploy-preview-2390--regen-marketplace.netlify.app/project/test-project123
2. Log in and create posts with different privacy settings then check the corresponding project data stream section to see the admin view.  As long as https://github.com/regen-network/regen-web/pull/2381 hasn't been merged yet, you should create posts from the corresponding deploy preview: https://deploy-preview-2381--regen-marketplace.netlify.app/

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
